chart/flux/templates/configmap-dashboards.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create -}}
```

Comment:
Why is this conditioned by RBAC?

Reply:
I think it was an accident, I'll remove it